ai/playbooks/service-wiring.md

```
@@ -0,0 +1,115 @@
+4. Copy the Client ID and Client Secret
+
+### Step 2 — Supabase dashboard (manual)
+
+1. Go to Authentication → Providers → Google
+2. Toggle on, paste Client ID and Client Secret from step 1
+
+### Step 3 — Environment variable (automated for `.env.local`; dashboard for Vercel)
+
+Add `google` to `NEXT_PUBLIC_AUTH_PROVIDERS`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)